### PR TITLE
Clean up pr 104954

### PR DESCRIPTION
**Pull request by Rageking8**
UI test filename typo fix and a simple comment rewording. Thanks.

CC @vincenzopalazzo (you spelled your name wrongly in the FIXME)

r? @estebank

### CODE REVIEW

File: src/test/ui/suggestions/suggest_print_over_printf.stderr [hunk 1]
```diff
@@ -1,5 +1,5 @@
 error[E0425]: cannot find function `printf` in this scope
-  --> $DIR/seggest_print_over_printf.rs:6:5
+  --> $DIR/seggest_print_over_printf.rs:5:5
```
Comment (compiler-errors) [inline src/test/ui/suggestions/suggest_print_over_printf.stderr:2]: needs bless

**Review by vincenzopalazzo — APPROVED** [review]
Thanks for typo check!

After the bless for me is good to go

File: src/test/ui/suggestions/suggest_print_over_printf.stderr (hunk 1 above)
Comment (Rageking8) [reply src/test/ui/suggestions/suggest_print_over_printf.stderr:2]: Yes, had to attend to something else when I found out of this error.

**Pull request merged**